### ISSUE TITLE
`runstatedir` is not set by autoconf

### ISSUE THREAD

**Issue by metalefty** [neutrinolabs/xrdp#2267]
After #2207, configure script relies on `runstatedir` but it is not set in autoconf 2.69.

`socketdir` and `sesmanruntimedir` are placed `/xrdp` and `/xrdp-sesman`.

I think a workaround like this is needed:
https://github.com/troglobit/sysklogd/issues/6
```
$ ./configure

xrdp will be compiled with:

  mp3lame                 no
  opus                    no
  fdkaac                  no
  jpeg                    no
  turbo jpeg              no
  rfxcodec                yes
  painter                 yes
  pixman                  no
  fuse                    no
  ipv6                    no
  ipv6only                no
  vsock                   no
  auth mechanism          PAM
  rdpsndaudin             no

  with imlib2             no

  development logging     no
  development streamcheck no

  strict_locations        no
  prefix                  /usr/local
  exec_prefix             ${prefix}
  libdir                  ${exec_prefix}/lib
  bindir                  ${exec_prefix}/bin
  sysconfdir              /etc
  localstatedir           /var
  runstatedir
  socketdir               /xrdp
  sesmanruntimedir        /xrdp-sesman

  unit tests performable  yes

  CFLAGS = -g -O2 -Wall -Wwrite-strings
  LDFLAGS =
```

**Comment by matt335672** [comment]
Good catch. I'll install autoconf 2.69 and have a look.

**Comment by metalefty** [comment]
The latest autoconf version available on FreeBSD is still 2.69.
